doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -32,22 +32,25 @@ Description of the Case
 -----------------------
 
 This example simulates the conveying of particles arranged in a plug/slug with a stationary layer.
-Since the initial particle layout plays an important role in the regime of the pneumatic conveying, the particles are already forming a plug at the beginning of the CFD-DEM simulation.
-Three simulation runs need to be performed.
-First, we call ``lethe-particles`` to load the particles with the file ``loading-particles.prm``.
-The setup of the simulation is a pipe with a solid object (mesh) that represents the shape of a slug.
-The insertion of particles is done using the plane insertion.
-Second, we call ``lethe-particles`` to settle the particles with the file ``settling-particles.prm``. Only the direction of gravity is changed.
-Finally, we call ``lethe-fluid-particles`` to simulate the dense pneumatic conveying with the file ``pneumatic-conveying.prm`` using periodic boundary conditions.
-We enable checkpointing in order to write the DEM checkpoint files which will be used as the starting point of the CFD-DEM simulation.
 The geometry of the pipe and the particle properties are based on the work of Lavrinec *et al*. [#lavrinec2021]_
 
+As the other Unresolved CFD-DEM simulations, this example starts from inserting the particles.
+
+However, since the initial position of the particles play an important role in the dynamics, we have an accomodation step to the case, in which particles are repositioned as shown in the following figure.
```

Comment:
```suggestion
However, since the initial position of the particles play an important role in the dynamics, a more involved insertion procedure is used to generate an initial plug of particles as illustrated in the following figure.
```